core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -48,6 +48,12 @@
     @ConfigItem(defaultValue = "false")
     public boolean addAllCharsets;
 
+    /**
+     * If all time zones should be added to the native image. This increases image size
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean includeAllTimeZones;
```

Comment:
Could we be consistent for the two config properties for charsets and time zones?

Reply:
Well I see two ways how to be consistent: (a) align with the native-image flag (my current proposal) or (b) align with other available properties.

(a) Might be less confusing for the people knowing the native-image flags.

I have no clear preference and I am ready to go with `all` instead of `include` if you and others prefer that.

Reply:
Yeah no OK, it makes perfect sense.